firmware/application/apps/ui_aprs_tx.hpp

```diff
@@ -82,6 +82,7 @@ class APRSTXView : public View {
 		6,
 		SymField::SYMFIELD_ALPHANUM
 	};
+	
```

Comment:
I am not sure if you could delete this file from the PR (I have an option but I am not sure of the consequences)